my_exe/CMakeLists.txt

```diff
@@ -3,11 +3,11 @@ target_link_libraries(my_exe PRIVATE project_options project_warnings) # link pr
 
 # Includes
 # because my_exe includes are private, it uses absolute paths.
-set(my_exe_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include")
+set(my_exe_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include" PARENT_SCOPE)
```

Comment:
`PARENT_SCOPE` does hit this variable in current scope!

better to use `CACHE PATH ...`

Reply:
Resolved in https://github.com/aminya/cpp_vcpkg_project/pull/5/commits/acd68ab9c6127a8c40086d243f15bd5e65c1162d